### PR TITLE
statsd and some more tests

### PR DESCRIPTION
**Pull request by ch40s3k**

### CODE REVIEW

File: alamo_scheduler/scheduler.py [hunk 1]
```diff
@@ -96,9 +112,10 @@ def consumer_messages(self):
 
             logger.debug(check)
             self.remove_job(check['id'])
-            enabled = check['triggers'][0]['enabled']
-            if enabled:
-                self.schedule_job(check)
+            for trigger in check['triggers']:
+                enabled = trigger['enabled']
```
Comment (pnuckowski) [inline alamo_scheduler/scheduler.py]: It should schedule specific check only once, no matter how many enabled triggers it has. `if any([t['enabled'] for t in triggers]): .....`

**Comment by pnuckowski** [comment]
+

File: alamo_scheduler/scheduler.py [hunk 2]
```diff
@@ -27,6 +29,17 @@ def __init__(self):
             settings.KAFKA__GROUP,
             settings.KAFKA__TOPIC
         )
+        self.statsd = self.initialize_statsd()
+
+    def initialize_statsd(self):
+        hostname = socket.getfqdn().replace('.', '_')
+        prefix = settings.STATSD__STATSD_PREFIX + '.{}'.format(hostname)
```
Comment (deejay1) [inline alamo_scheduler/scheduler.py]:
```python
prefix = '{}.{}'.format(settings.STATSD__STATSD_PREFIX, hostname)
```

Comment (ch40s3k) [inline alamo_scheduler/scheduler.py]: Done

**Comment by deejay1** [comment]
:+1:

**Pull request merged**